<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>421</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Middle Stream</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="middleLine">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>1,2; 1,5; 1,3 </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="middleUnit">
          <item>
           <property name="text">
            <string>kg/h</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>m/s</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>g/s</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Inner Stream  </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="innerLine">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>1,2; 1,5; 1,3 </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="innerUnit">
          <item>
           <property name="text">
            <string>kg/h</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>m/s</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>g/s</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Outer Stream </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="outerLine">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>1,2; 1,5; 1,3 </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="outerUnit">
          <item>
           <property name="text">
            <string>kg/h</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>m/s</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>g/s</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="CreateFile">
          <property name="text">
           <string>Select folder to create file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ChoseFile">
          <property name="text">
           <string>Chose existing file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currentFile">
          <property name="maximumSize">
           <size>
            <width>415</width>
            <height>68</height>
           </size>
          </property>
          <property name="text">
           <string>Plese select file or folder</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
